docs/examples/externalsocket.c

```diff
@@ -157,10 +159,16 @@ int main(void)
 
     curl_easy_cleanup(curl);
 
+    close(sockfd);
```

Comment:
easy cleanup closes the socket, i believe that happens in all cases

Reply:
Well, then the close callback gets called since `CURLOPT_CLOSESOCKETFUNCTION` is used by the example, which (for some reason) doesn't close the socket! So I think adding a close is ok.

But doesn't windows use `closesocket()` for this?

Reply:
> Well, then the close callback gets called since `CURLOPT_CLOSESOCKETFUNCTION` is used by the example, which (for some reason) doesn't close the socket! So I think adding a close is ok.

ah ok. maybe that makes more sense since it's opened externally.

> But doesn't windows use closesocket() for this?

close maps to closesocket in the example so it's fine.